<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryDetailDialog</class>
 <widget class="QDialog" name="EntryDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="8" column="1">
       <widget class="QPlainTextEdit" name="remarksEdit">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameEdit">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>标识(*)：</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>账户名：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>确认密码：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="passwordEdit">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="usernameEdit">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="confirmEdit">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>备注：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>分组：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="otherNameEdit">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="maxLength">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>别名：</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="groupEdit">
        <property name="font">
         <font>
          <family>等线</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>passwordEdit</zorder>
     <zorder>remarksEdit</zorder>
     <zorder>nameEdit</zorder>
     <zorder>label</zorder>
     <zorder>label_3</zorder>
     <zorder>label_5</zorder>
     <zorder>label_4</zorder>
     <zorder>usernameEdit</zorder>
     <zorder>confirmEdit</zorder>
     <zorder>label_7</zorder>
     <zorder>label_6</zorder>
     <zorder>otherNameEdit</zorder>
     <zorder>label_2</zorder>
     <zorder>groupEdit</zorder>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="OKButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>确认</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>otherNameEdit</tabstop>
  <tabstop>usernameEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>confirmEdit</tabstop>
  <tabstop>groupEdit</tabstop>
  <tabstop>remarksEdit</tabstop>
  <tabstop>OKButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
